app/uk/gov/hmrc/incometaxsubscriptionselfemployedfrontend/views/agent/SelfEmployedCYA.scala.html

```diff
@@ -0,0 +1,144 @@
+@*
```

Comment:
I didn't find units test for this page `SelfEmployedCYAViewSpec`.

Reply:
Done.